<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <arg name="node_name" default="m8p_rover"/>
  <arg name="param_file_name" default="c94_m8p_rover" />
  <arg name="output" default="screen" />
  <arg name="respawn" default="true" />
  <arg name="respawn_delay" default="30" />
  <arg name="clear_params" default="true" />

  <!-- <node pkg="ublox_gps" type="ublox_gps" name="$(arg node_name)"
        output="$(arg output)" 
        clear_params="$(arg clear_params)"
        respawn="$(arg respawn)" 
        respawn_delay="$(arg respawn_delay)">
    <rosparam command="load" 
              file="$(find ioniq_control)/config/ublox/$(arg param_file_name).yaml" />
  </node> -->

  <node pkg="socketcan_bridge" type="socketcan_bridge_node" name="gcan_node" ns="gcan">
    <param name="can_device" type="str" value="can0" />
    <remap from="sent_messages" to="sent_msgs"/>
    <remap from="received_messages" to="recv_msgs"/>
  </node>
  <!-- <node pkg="socketcan_bridge" type="socketcan_bridge_node" name="ccan">
    <param name="can_device" type="string" value="can1" />
  </node> -->

  <node pkg="ioniq_control" type="ioniq_control_node" name="ioniq_control_node" output="screen">
    <param name="path_file_name" type="str" value="$(find ioniq_control)/path/PATH.txt"/>
  </node>

</launch>
